frontend/webapp/hooks/overview/useMetrics.ts

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import { useQuery } from '@apollo/client';
+import { GET_METRICS } from '@/graphql/mutations/metrics';
+import type { ActualDestination, K8sActualSource, OverviewMetricsResponse } from '@/types';
+
+export function useMetrics({ sources, destinations }: { sources: K8sActualSource[]; destinations: ActualDestination[] }) {
+  const { data, refetch } = useQuery<OverviewMetricsResponse>(GET_METRICS);
+
+  // this is just to re-render the mockup data
+  const [_, setCount] = useState(0);
+
+  useEffect(() => {
+    const interval = setInterval(async () => {
+      await refetch();
+      setCount((n) => n + 1);
+    }, 1000);
```

Comment:
double check to pulling time with Ron, from what I remember the backend collect the metrics in an interval of 5sec or 10s